tests/fuzz/run.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+import argparse
+import logging
+import os
+import re
+import subprocess
+
+from concurrent.futures import ProcessPoolExecutor, as_completed
+
+
+def job(command):
+    """Takes a list of str, and runs it as a subprocess."""
```

Comment:
what does a 'list of str' mean?
```suggestion
    """Takes a list of command strings??, and runs it as a subprocess."""
```

Reply:
`str` is the string type in Python, and most Python commands treating shell commands take it as `["path_to_bin", "--param-a", "param-b"]` so i did the same